languages/en-CA.js

```diff
@@ -1,15 +1,173 @@
+				guildsAdded.length ? `**Guilds Added**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**Guilds Removed**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: 'Evaluates arbitrary Javascript. Reserved for bot owner.',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'The eval command evaluates code as-in, any error thrown from it will be handled.',
+				'It also uses the flags feature. Write --silent, --depth=number or --async to customize the output.',
+				'The --silent flag will make it output nothing.',
+				"The --depth flag accepts a number, for example, --depth=2, to customize util.inspect's depth.",
+				'The --async flag will wrap the code into an async function where you can enjoy the use of await, however, if you want to return something, you will need the return keyword.',
+				'The --showHidden flag will enable the showHidden option in util.inspect.',
+				'If the output is too large, it\'ll send the output as a file, or in the console if the bot does not have the ATTACH_FILES permission.'
+			].join('\n'),
+			COMMAND_EVAL_ERROR: (time, output, type) => `**Error**:${output}\n**Type**:${type}\n${time}`,
+			COMMAND_EVAL_OUTPUT: (time, output, type) => `**Output**:${output}\n**Type**:${type}\n${time}`,
+			COMMAND_EVAL_SENDFILE: (time, type) => `Output was too long... sent the result as a file.\n**Type**:${type}\n${time}`,
+			COMMAND_EVAL_SENDCONSOLE: (time, type) => `Output was too long... sent the result to console.\n**Type**:${type}\n${time}`,
+			COMMAND_RELOAD: (type, name, time) => `✅ Reloaded ${type}: ${name}. (Took: ${time})`,
+			COMMAND_RELOAD_FAILED: (type, name) => `❌ Failed to reload ${type}: ${name}. Please check your Console.`,
+			COMMAND_RELOAD_ALL: (type, time) => `✅ Reloaded all ${type}. (Took: ${time})`,
+			COMMAND_RELOAD_EVERYTHING: (time) => `✅ Reloaded everything. (Took: ${time})`,
+			COMMAND_RELOAD_DESCRIPTION: 'Reloads a klasa piece, or all pieces of a klasa store.',
+			COMMAND_REBOOT: 'Rebooting...',
+			COMMAND_REBOOT_DESCRIPTION: 'Reboots the bot.',
+			COMMAND_ENABLE: (type, name) => `+ Successfully enabled ${type}: ${name}`,
+			COMMAND_ENABLE_DESCRIPTION: 'Re-enables or temporarily enables a command/inhibitor/monitor/finalizer. Default state restored on reboot.',
+			COMMAND_DISABLE: (type, name) => `+ Successfully disabled ${type}: ${name}`,
+			COMMAND_DISABLE_DESCRIPTION: 'Re-disables or temporarily disables a command/inhibitor/monitor/finalizer/event. Default state restored on reboot.',
+			COMMAND_DISABLE_WARN: 'You probably don\'t want to disable that, since you wouldn\'t be able to run any command to enable it again',
+			COMMAND_CONF_NOKEY: 'You must provide a key',
+			COMMAND_CONF_NOVALUE: 'You must provide a value',
+			COMMAND_CONF_GUARDED: (name) => `${util.toTitleCase(name)} may not be disabled.`,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['util' is not defined.](https://app.codacy.com/manual/Butterstroke/MargarineBot/pullRequest?prid=5457955)